### PR TITLE
add ingress/router nodeType to pilot_cli

### PR DESCRIPTION
**Pull request by quanjielin**

### CODE REVIEW

File: pilot/tools/debug/pilot_cli.go [hunk 1]
```diff
@@ -182,6 +189,7 @@ func main() {
 	appName := flag.String("app", "", "app label. Should be set if pod name is not provided. It will be used to find "+
 		"the pod that has the same app label. Ignored if --pod is set.")
 	podIP := flag.String("ip", "", "pod IP. If omit, pod IP will be found from registry.")
+	nodeType := flag.String("nodetype", "", "sidecar, ingress, router. Default sidecar.")
```
Comment (diemtvu) [inline pilot/tools/debug/pilot_cli.go]: "" -> "sidecar"

File: pilot/tools/debug/pilot_cli.go [hunk 2]
```diff
@@ -115,10 +116,16 @@ func (p *PodInfo) populatePodNameAndIP(kubeconfig string) bool {
 }
 
 func (p PodInfo) makeNodeID() string {
+	if p.NodeType != "" {
```
Comment (diemtvu) [inline pilot/tools/debug/pilot_cli.go]: You can remove this, if node type actually "sidecar" by default.

File: pilot/tools/debug/pilot_cli.go (hunk 1 above)
Comment (quanjielin) [reply pilot/tools/debug/pilot_cli.go]: done.

File: pilot/tools/debug/pilot_cli.go (hunk 2 above)
Comment (quanjielin) [reply pilot/tools/debug/pilot_cli.go]: done.

**Comment by quanjielin** [comment]
Close this PR as https://github.com/istio/istio/pull/5531 took the change, thanks @diemtvu.

**Pull request closed without merging**